crates/sui-open-rpc/spec/openrpc.json

```diff
@@ -5317,6 +5317,28 @@
       "BigInt_for_uint64": {
         "type": "string"
       },
+      "Bn254FqElement": {
+        "description": "A struct that stores a Bn254 Fq field element as 32 bytes.",
+        "type": "array",
+        "items": {
+          "type": "integer",
+          "format": "uint8",
+          "minimum": 0.0
+        },
+        "maxItems": 32,
+        "minItems": 32
+      },
+      "Bn254FrElement": {
+        "description": "A struct that stores a Bn254 Fr field element as 32 bytes.",
+        "type": "array",
+        "items": {
+          "type": "integer",
+          "format": "uint8",
+          "minimum": 0.0
+        },
+        "maxItems": 32,
+        "minItems": 32
+      },
```

Comment:
This doesn't look right. The JsonSchema should match the serialization format which is a String and not an array of [u8; 32]